scripts/istat/geos/import_config.txt

```diff
@@ -0,0 +1,41 @@
+import: {
```

Comment:
Could you use `//` to comment out the municipal import to make it clear that we are not yet ingesting that? Also, we can combine the two others into "istat_geos".

Reply:
I commented the municipal import, but I didn't combine the two others since I did them in two separate imports. Do you want me reorganize them and import these two together?